src/pages/signIn/signUp.jsx

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios';
+import { Formik } from 'formik';
+import React, { useState } from 'react';
+import { useQuery } from 'react-query';
+import { useDispatch } from 'react-redux';
+import * as Yup from 'yup';
+
+import { serverPath } from '../../constants/noteList';
+import { getUserInfo } from '../../redux/actions/actionCreators';
+import { SignForm, ValidContainer, StyledField, ErrorTip, SignInButton, RegularText, StyledLink, StyledTitle } from './styled';
+
+const ValidScheme = Yup.object().shape({
+    email: Yup.string().email('Please use @ and . for adding email').required('Please enter Email'),
+    password: Yup.string()
+        .min(3, 'password must contain at least 3 symblos')
+        .required('Please enter password'),
+    firstName: Yup.string().required('Please enter your name'),
+    lastName: Yup.string().required('Enter your last name'),
+    dateOfBirth: Yup.string().required('Please pick date of birth'),
+    confirmPassword: Yup.string().oneOf([Yup.ref('password'),null],'Passwords must match ')
+});
+const SignUp = () => {
+    const [newUser, setNewUser] = useState(null);
+    const dispatch=useDispatch()
+    const sendDataToServer = async (data) => {
+        const resp = await axios.post(serverPath, data);
+        dispatch(getUserInfo(data))
+        setNewUser(null)
+        return resp.status;
+    };
+    useQuery(
```

Comment:
please read the react-query documentation. query is only for GET requests,  use mutations

Reply:
Done